frontend/src/components/Post/Post.js

```diff
@@ -1,16 +1,18 @@
 import '../../config';
 import axios from "axios";
 import { Link } from "react-router-dom";
-import React, { useState, useEffect } from 'react'
-
-import timeElapsedSincePosted from "../../services/TimeService";
+import React, { useState, useEffect } from 'react';
 
+// Project
 import CommentList from "../Comment/CommentList";
 import { PostComment } from "../Comment/PostComment";
+import FollowingButton from "../Following/FollowingButton.js";
+import timeElapsedSincePosted from "../../services/TimeService";
 
+// Stylesheets
 import "./Post.scss";
 
-export default function Post(props) {
+export default function Post(props){
```

Comment:
I think you should keep the space (there is a space before the bracket in our Coding Standards example)